src/panels/lovelace/cards/energy/hui-energy-usage-graph-card.ts

```diff
@@ -598,15 +650,16 @@ export class HuiEnergyUsageGraphCard
 
     Object.entries(combinedData).forEach(([type, sources]) => {
       Object.entries(sources).forEach(([statId, source], idx) => {
+        const overrideColor = colors[type].overrides?.[idx];
         const modifiedColor =
```

Comment:
We should not calculate a `modifiedColor` if we have a `overrideColor`, as we will not use it anyway